package.json

```diff
@@ -10,6 +10,9 @@
     "url": "https://github.com/readmeio/syntax-highlighter.git"
   },
   "homepage": "https://readmeio.github.io/syntax-highlighter/",
+  "engines": {
+    "node": "^14 || ^16"
```

Comment:
Since we're introducing React 17 support in this we might as well make this a major release and formally deprecate Node 12 support while we're at it.

Reply:
That sounds about right to me!